lib/Optimizer/GraphOptimizer/FunctionPassManager.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) Glow Contributors. See CONTRIBUTORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Optimizer/GraphOptimizer/FunctionPassManager.h"
+#include "glow/Optimizer/GraphOptimizer/FunctionPasses.h"
+
+#include <glog/logging.h>
+
+using namespace glow;
+
+using ThePassManager = FunctionPassManager;
```

Comment:
This is an excellent question. 

>  is the intention to make the FPM and IRFPM look as similar as possible?

Yes, the main intention is too keep FPM and IRFPM look as similar as possible, so that you can copy&paste any changes or updates from one into another in most cases.

Reply:
Is it very annoying? If it is, I could use the actual type names instead of aliases.

Reply:
Not a huge deal. Perhaps it's the way I read code, but I was scanning through things/skipped over the `using` here, and then was confused as to why it wasn't `FunctionPassManager`/`Function` being used. Then saw the `using` here and figured it was just stylistic. Maybe worth adding a small disclaimer above `using ThePassManager` here just so it's clear the motivation.

Reply:
> Maybe worth adding a small disclaimer above using ThePassManager here just so it's clear the motivation.

Good idea. I'll add such a disclaimer.